features/hydra/entrypoint.feature

```diff
@@ -9,6 +9,7 @@ Feature: Entrypoint support
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the header "Content-Location" should not exist
```

Comment:
shouldn't we add the test on an actual resource, the entrypoint has its own container

Reply:
Moved from `features/hydra/entrypoint.feature` to `features/main/crud.feature`